be/src/exec/workgroup/work_group.h

```diff
@@ -272,6 +279,17 @@ class WorkGroupManager {
 
     std::unique_ptr<WorkerOwnerManager> _driver_worker_owner_manager;
     std::unique_ptr<WorkerOwnerManager> _scan_worker_owner_manager;
+
+    std::once_flag init_metrics_once_flag;
+    std::unordered_map<std::string, int128_t> _wg_metrics;
+
+    std::unordered_map<std::string, std::unique_ptr<starrocks::DoubleGauge>> _wg_cpu_limit_metrics;
```

Comment:
please use work group id, not work group name as map key

Reply:
The same group name may have different ids, for example, delete and then create, but it is the same resource group for users, and metrics are also identified by name.So I think it is batter to use the name and will update the latest id of the name in _wg_metrics. Or is it bad to do this in c++ ?

Reply:
Ok, we could use string as key.

```
is it bad to do this in c++ 
```
integer performance is better than string when as map key.  But there shouldn't many work groups in a cluster, so, use work group name as key maybe also OK.